.github/scripts/changelog_checker

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
+
+set -ue
+
+UNTRACKED="
+appveyor_build.cmd
+appveyor.patch
+appveyor_test.sh
+appveyor.yml
+AUTHORS
+CONTRIBUTING.md
+.gitattributes
+.github
+.gitignore
+ISSUE_TEMPLATE.md
+.ocamlinit
+.ocp-indent
+.ocplint
+README.md
+.travis-ci.sh
+.travis.yml
+master_changes.md
+"
+changelog=master_changes.md
+diffile=/tmp/diff
+
+git fetch origin $GITHUB_BASE_REF --depth=1 --quiet
+echo "> base commit"
+git show origin/$GITHUB_BASE_REF --format=oneline -s
+
+git diff origin/$GITHUB_BASE_REF --name-status --diff-filter=AM > $diffile
+updated=0
+if [ -z "`grep -s $changelog $diffile`" ] ; then
```

Comment:
`if ! grep -sq $changelog $diffile; then ...`
Or even just `grep -sq $changelog $diffile || updated=1` actually, but that's less readavle

Reply:
Thanks!